Updated references to content view environments (#5080)
theforeman/foreman-documentation: guides/common/modules/ref_best-practices-for-errata.adoc

* Use errata to add patches for security issues to a frozen set of content without unnecessarily updating other unaffected packages.
* Automate errata management by using a Hammer script or an {ansible-docs-url}[Ansible Playbook].

> * Check the host details page to compare errata between the current content view environments and the Library lifecycle environment, which contains the latest synchronized packages.

Reviewer: ```suggestion
* Check the host details page to compare errata between the current content view environments and the Library content view environment, which contains the latest synchronized packages.
```

> * View the errata on the host details page and compare the errata of the current content view environments to the Library lifecycle environment, which contains the latest synchronized packages.

Reviewer 2: The added "the" changes the meaning slightly and I don't think it's correct here.

In my reading, 
"View errata on the host details page" means "The host details page is where you can see (all or unspecified) errata"
"View the errata on the host details page" means "Look at just the selection of errata that is shown at the host details page".

I think the first meaning is correct, but the latter is not.

It is a very small difference though and I might be off, or the intention might be correct. @aneta-petrova do you have an opinion please?

Reviewer 3: I'd say that between @jberendz, @jafiala, and myself, I'm the least knowledgeable person to advise on proper article usage :smile: I suppose there is a difference in these sentences depending on whether the article is or isn't present but... are these nuances something we should expect users to notice or even be aware of?

So, because I'm not sure what the answer is, my mind went to: Can this sentence be simplified in a way that would make it easier to understand to anyone, regardless of their background and level of knowledge?

```suggestion
* Check the host details page to compare errata between the current content view environments and the Library lifecycle environment, which contains the latest synchronized packages.
```
